projects/fsastorefrontlib/src/cms-components/my-account/update-profile/update-profile.component.ts

```diff
@@ -33,6 +34,9 @@ export class FSUpdateProfileComponent extends UpdateProfileComponent
   subscription = new Subscription();
 
   ngOnInit() {
+    this.form
+      .get('dateOfBirth')
+      .setValidators([Validators.min(1900), Validators.max(2100)]);
```

Comment:
Hm, dateofbirth should be in the past above 1900...Why is max 2100?

Reply:
There is also FSUpdateProfileComponentService where form is being created, move these validations there

Reply:
In the task ticket is mentioned min and max values for date fields. Ok I'll move it to service

Reply:
> Hm, dateofbirth should be in the past above 1900...Why is max 2100?

I checked with Sofi and there is no need for maxValue validation. I'll add one more validation `dateOfBirthValidator`